docs/_articles/en/apex/apex-tasks.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Apex Tasks
+lang: en
+---
+
+This repository contains sample Apex tasks that you can complete in Code Builder or VS Code Extensions for desktop. We constantly add to this list, so make sure you visit this topic often. 
+
+
+## Use Anonymous Apex to Mass Update Account Ratings
+
+Say you have accounts with `Rating` set to `Cold` because of unfavorable business conditions. You now want to make mass updates to these accounts and change their rating to `Warm`.  Say you want to limit this update to mid-sized accounts with more than a certain amount of revenue. 
+
+This Anonymous Apex code updates ratings of accounts that have an annual revenue greater than `$10,000,000` and have more than `1000` employees:
```

Comment:
```suggestion
This anonymous Apex code updates ratings of accounts that have an annual revenue greater than `$10,000,000` and have more than `1000` employees:
```

Reply:
See https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_anonymous_block.htm in which we don't capitalize "anonymous" when used within a sentence.  Might as well keep it consistent :)

Reply:
We've gone back and forth about this. I'll make updates in other topics also.

Reply:
Also, can you reorg so that the "This anonymous Apex code..." sentence comes right before the code?  Right now it's coming before the numbered list, so the colon looks odd.  Maybe you can keep the sentence where it is and just say:

The anonymous Apex code updates ratings of accounts that have an annual revenue greater than `$10,000,000` and have more than `1000` employees.

Or move the sentence to right _below_ the code.